src/use_case/chat/ChatInteractor.java

```diff
@@ -2,29 +2,34 @@
 
 import entities.rooms.Message;
 import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import use_case.rooms.MessageDataAccessInterface;
 
 public class ChatInteractor implements ChatInputBoundary {
   final ChatOutputBoundary outputBoundary;
-  final ChatMessageDataAccessInterface dataAccessInterface;
+  final MessageDataAccessInterface dataAccessInterface;
   final ChatUserDataAccessInterface chatUserDataAccessInterface;
 
   public ChatInteractor(
       ChatOutputBoundary outputBoundary,
-      ChatMessageDataAccessInterface dataAccessInterface,
+      MessageDataAccessInterface dataAccessInterface,
       ChatUserDataAccessInterface chatUserDataAccessInterface) {
     this.outputBoundary = outputBoundary;
     this.dataAccessInterface = dataAccessInterface;
     this.chatUserDataAccessInterface = chatUserDataAccessInterface;
   }
 
   public void loadAllMessages() {
-    var messages = dataAccessInterface.getAllMessages();
+    // TODO: Delete this function
+    List<Message> messages = new ArrayList<>();
     outputBoundary.presentMessages(messages);
   }
 
   public void sendMessage(String messageText) {
+    // TODO: Delete this function
     Message message =
-        new Message(Instant.now(), messageText, chatUserDataAccessInterface.get().getUid());
-    dataAccessInterface.save(message);
+        new Message(Instant.now(), messageText, chatUserDataAccessInterface.get().getEmail());
+    // dataAccessInterface.save(message);
```

Comment:
why did this change? (I'm assuming from old issues with getting the build to work)